pkg/interface/src/logic/state/base.ts

```diff
@@ -112,8 +115,12 @@ export const createState = <T extends {}>(
   name: string,
   properties: T | ((set: SetState<T & BaseState<T>>, get: GetState<T & BaseState<T>>) => T),
   blacklist: (keyof BaseState<T> | keyof T)[] = [],
-  subscriptions: ((set: SetState<T & BaseState<T>>, get: GetState<T & BaseState<T>>) => SubscriptionRequestInterface)[] = []
+  subscriptions: ((set: SetState<T & BaseState<T>>, get: GetState<T & BaseState<T>>) => SubscriptionRequestInterface)[] = [],
+  clear?: Partial<T>
```

Comment:
I think it's a little confusing that the empty fields themselves are called clear and the function to use them is called clear.